src/main/java/uk/gov/pay/products/persistence/entity/ProductEntity.java

```diff
@@ -48,6 +48,9 @@
     @Column(name = "service_name")
     private String serviceName;
 
+    @Column(name = "friendly_url")
+    private String friendlyUrl;
```

Comment:
Although in Product what we return should be the full friendly URL, what is held here in the entity is not the full friendly URL but merely a portion of it hence why we should probably settle for a different naming.